src/main/java/com/onarandombox/MultiverseCore/commands/CreateCommand.java

```diff
@@ -0,0 +1,110 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+import co.aikar.commands.CommandIssuer;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Flags;
+import co.aikar.commands.annotation.Optional;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.commandtools.flags.FlagGroup;
+import com.onarandombox.MultiverseCore.commandtools.flags.MVFlag;
+import com.onarandombox.MultiverseCore.commandtools.flags.MVValueFlag;
+import com.onarandombox.MultiverseCore.commandtools.flags.ParsedFlags;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.WorldType;
+import org.bukkit.command.CommandException;
+import org.bukkit.plugin.Plugin;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class CreateCommand extends MultiverseCommand {
+
+    public CreateCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+
+        this.plugin.getCommandManager().getFlagsManager().registerFlagGroup(FlagGroup.builder("mvcreate")
```

Comment:
Maybe `MultiverseCommand` needs a protected shortcut method for registering a flag group.

Reply:
yea, i prob adding wordmanager shortcut also, was just "makeitwork" first